demos/search.js

```diff
@@ -0,0 +1,613 @@
+		"Siberian Thrush" : "Zoothera sibirica",
+		"Eyebrowed Thrush" : "Turdus obscurus",
+		"Dusky Thrush" : "Turdus eunomus",
+		"Black-throated Thrush" : "Turdus atrogularis",
+		"Pallas`s Grasshopper Warbler" : "Locustella certhiola",
+		"Spectacled Warbler" : "Sylvia conspicillata",
+		"Two-barred Warbler" : "Phylloscopus plumbeitarsus",
+		"Eastern Bonelli’s Warbler" : "Phylloscopus orientalis",
+		"Collared Flycatcher" : "Ficedula albicollis",
+		"Wallcreeper" : "Tichodroma muraria",
+		"Turkestan Shrike" : "Lanius phoenicuroides",
+		"Steppe Grey Shrike" : "Lanius pallidirostris",
+		"Spanish Sparrow" : "Passer hispaniolensis",
+		"Red-eyed Vireo" : "Vireo olivaceus",
+		"Myrtle Warbler" : "Dendroica coronata",
+		"White-crowned Sparrow" : "Zonotrichia leucophrys",
+		"White-throated Sparrow" : "Zonotrichia albicollis",
+		"Cretzschmar`s Bunting" : "Emberiza caesia",
+		"Chestnut Bunting" : "Emberiza rutila",
+		"Red-headed Bunting" : "Emberiza bruniceps",
+		"Black-headed Bunting" : "Emberiza melanocephala",
+		"Indigo Bunting" : "Passerina cyanea",
+		"Balearic Woodchat Shrike" : "Lanius senator badius",
+		"Demoiselle Crane" : "Grus virgo",
+		"Chough" : "Pyrrhocorax pyrrhocorax",
+		"Red-Billed Chough" : "Pyrrhocorax graculus",
+		"Elegant Tern" : "Sterna elegans",
+		"Chukar" : "Alectoris chukar",
+		"Yellow-Billed Cuckoo" : "Coccyzus americanus",
+		"American Sandwich Tern" : "Sterna sandvicensis acuflavida",
+		"Olive-Tree Warbler" : "Hippolais olivetorum",
+		"Eastern Olivaceous Warbler" : "Acrocephalus pallidus",
+		"Indian Cormorant" : "Phalacrocorax fuscicollis",
+		"Spur-Winged Lapwing" : "Vanellus spinosus",
+		"Yelkouan Shearwater" : "Puffinus yelkouan",
+		"Trumpeter Finch" : "Bucanetes githagineus",
+		"Red Grouse" : "Lagopus scoticus",
+		"Rock Ptarmigan" : "Lagopus mutus",
+		"Long-Tailed Cormorant" : "Phalacrocorax africanus",
+		"Double-crested Cormorant" : "Phalacrocorax auritus",
+		"Magnificent Frigatebird" : "Fregata magnificens",
+		"Naumann's Thrush" : "Turdus naumanni",
+		"Oriental Pratincole" : "Glareola maldivarum",
+		"Bufflehead" : "Bucephala albeola",
+		"Snowfinch" : "Montifrigilla nivalis",
+		"Ural owl" : "Strix uralensis",
+		"Spanish Wagtail" : "Motacilla iberiae",
+		"Song Sparrow" : "Melospiza melodia",
+		"Rock Bunting" : "Emberiza cia",
+		"Siberian Rubythroat" : "Luscinia calliope",
+		"Pallid Swift" : "Apus pallidus",
+		"Eurasian Pygmy Owl" : "Glaucidium passerinum",
+		"Madeira Little Shearwater" : "Puffinus baroli",
+		"House Finch" : "Carpodacus mexicanus",
+		"Green Heron" : "Butorides virescens",
+		"Solitary Sandpiper" : "Tringa solitaria",
+		"Heuglin's Gull" : "Larus heuglini"
+	};
+	function searchResponse( term ) {
+		term = term.toLowerCase();
+		var result = [];
+		for ( var key in database ) {
+			if ( key.toLowerCase().indexOf( term ) !== -1 ) {
+				result.push( {
+					id: database[key],
+					label: key,
+					value: key
+				} );
+			}
+		}
+		return result;
+	}
```

Comment:
I had left this out on the assumption that perhaps the previous limit was there as safety measure for the PHP server, and that returning the data as-is would better demonstrate how jQuery UI handles this. However, I neglected to check this after I changed it, and didn't realize it had no default limit. It seems it has neither a numerical limit, nor a visual limit to prevent scroll overflow beyond the browser's viewport.

I've restored it.